app/controllers/concerns/wordpress_embeddable.rb

```diff
@@ -26,7 +26,7 @@ def update_template
 
     base_uri = Rails.configuration.x.wordpress_template[:base_uri][get_division_from_url]
 
-    WordpressTemplate.update(
+    ::WordpressTemplate.update(
```

Comment:
Didn't we determine that this change was unnecessary?

Reply:
not really but I'll make the change in this PR